.github/workflows/add_backport_label.yml

```diff
@@ -1,8 +1,7 @@
 name: Add Backport Label
-
 on:
-  branches: main
-  pull_request:
+  pull_request_target:
+    branches: main
     types: opened
```

Comment:
Can we change the types to `closed` or else we won't be able to open backport PRs for closed PR?

Reply:
Also, we can add `labeled` as a type too.
```
types:
   closed
   labeled
```

Reply:
> Can we change the types to `closed` or else we won't be able to open backport PRs for closed PR?

This is just the workflow to add a label automatically. It doesn't prevent us from adding the label ourselves manually.

Reply:
Although looking around, seems we might want to replace all this with the `labeler` workflow.  See https://github.com/opensearch-project/opensearch-plugins/blob/main/WORKFLOWS.md#integration-1

@ryanbogan WDYT?

Reply:
Looks like that is for labeling based on the modified files.  I'm guessing there would be an option to just add the same label to everything, which would accomplish the same goal.

Reply:
@ryanbogan We can always make `*` the modified files.  Anyway, let's keep the current one for just this one but in the future if we find we need to do more auto-labeling let's consider migrating since it's one workflow but possibly multiple labels.